hack/build-go.sh

```diff
@@ -30,5 +28,19 @@ fi
 
 mkdir -p ${BIN_PATH}
```

Comment:
nit: give some hint to the user of the script.

```suggestion
mkdir -p ${BIN_PATH}
echo "Building binaries into ${BIN_PATH} ..."
```

Reply:
I don't see a reason why not make the script more user friendly. Done :raised_hands:

Reply:
Current output:

```
$ make
hack/build-go.sh
Using version from git...
Building binaries into _output/linux/amd64
Building github.com/openshift/cluster-version-operator cluster-version-operator-tests binary (v1.0.0-1371-gb2af387c)
Compressing the cluster-version-operator-tests binary
Building github.com/openshift/cluster-version-operator cluster-version-operator binary (v1.0.0-1371-gb2af387c)
```